### PR TITLE
[#133759955] Feature: Add `friend_origin` to Reward webhook

### PR DESCRIPTION
**Pull request by Tonkonozhenko**

### CODE REVIEW

File: source/web_hooks/reward.rst [hunk 1]
```diff
@@ -49,6 +49,26 @@ it is getting paid after Referral is approved.
   * **coupon_code** — coupon code received by person as a reward (null when
     **incentive_type** is rebate or other)
 
+* **advocate_reward** - subhash of data related to the advocate event
```
Comment (zhuravel) [inline source/web_hooks/reward.rst]: advocate_reward → advocate_origin
friend_reward → friend_origin

**Comment by zhuravel** [comment]
@Tonkonozhenko Please fix Travis build.

**Comment by Tonkonozhenko** [comment]
@zhuravel I've just done it.

File: source/web_hooks/reward.rst [hunk 2]
```diff
@@ -49,6 +49,39 @@ it is getting paid after Referral is approved.
...
+* **advocate_origin** - subhash of data related to the advocate event
+
+  * **type**
+
+    * *"Purchase"* for post-purchase campaign
+    * *"AffiliateMember"* for standalone campaign
+
+  * **id** — unique identifier of the advocate's origin event
+
+   *For Affiliate Member*
```
Comment (zhuravel) [inline source/web_hooks/reward.rst]: Either add a colon here or remove in `For Purchase:`

File: source/web_hooks/reward.rst [hunk 3]
```diff
@@ -49,6 +49,39 @@ it is getting paid after Referral is approved.
...
+
+* **friend_origin** - subhash of data related to the friend event
+
+  * **type**
+
+    * *"Purchase"* for post-purchase campaign
+    * *"Event"* for custom campaign
+
+  * **id** — unique identifier of the referred origin
+  * **order_number** - unique identifier of referred order
```
Comment (zhuravel) [inline source/web_hooks/reward.rst]: I prefer descriptions to match, let's change this to "unique identifier of friend's order"

**Pull request merged**